### DISCUSSION TITLE
Jupyter Run: running flyte code directly from a jupyter notebook

### DISCUSSION THREAD

**Discussion by cosmicBboy** [flyteorg/flyte#3878] [RFC Incubator]
### User Story

As a data scientist, I want to run flytekit code directly from within a notebook so that I don't have to go through the entire `.py` -> `pyflyte register` loop from a CLI.

See proof of concept PR: https://github.com/flyteorg/flytekit/pull/1742/files
Example notebook: https://github.com/flyteorg/flytekit/blob/40fd938af11ca1e8152210dfe2675c097669630b/workflow.ipynb

**Comment by fg91** [comment]
Hey @cosmicBboy ,

I'm currently sorting through RFC incubator discussions that we haven't talked about in a while. What is the status here, have you continued to work on this?

**Reply by cosmicBboy** [reply]
We haven't moved on this beyond the POC PR above. Still not convinced the POC UX is the right one though.

**Comment by davidmirror-ops** [comment]
2023-11-09 Contributor's meetup notes: this entry is better suited for an Issue. @cosmicBboy, feel free to proceed creating a new Issue.

**Comment by davidmirror-ops** [comment]
This is coming thanks to @Mecoli1219 et.al.
